api/src/pcapi/core/bookings/repository.py

```diff
@@ -323,6 +323,22 @@ def get_bookings_from_deposit(deposit_id: int) -> list[Booking]:
     )
 
 
+def get_validated_bookings_by_offer(
+    offer_id: int, export_type: BookingExportType | None = BookingExportType.CSV
```

Comment:
Pourquoi accepter `None` comme valeur possible de `export_type` ?

```suggestion
    offer_id: int, export_type: BookingExportType = BookingExportType.CSV
```